appveyor/deploy.ps1

```diff
@@ -17,8 +15,8 @@ function deploy_to_bintray() {
     $username_password = "${env:bintray_username}:${env:bintray_api_key}"
 
     Write-Host "Uploading: $filepath"
-    Write-Host "to https://api.bintray.com/content/hgomersall/generic/PyFFTW-development-builds/$short_version/$filename"
```

Comment:
I think we still need a short version (e.g. 0.10.0) for the path, otherwise every dev release will produce a new path. This can be extracted from the version string quite easily I think (split on + for dev releases). What do you think? Happy to change my mind on this.